src/commands/start.ts

```diff
@@ -9,9 +10,19 @@ import { getDebugFunction, setLogLevelByName } from '../utils/logger';
 import { ExternalCliOptions, sharedCliOptions } from './shared';
 import { CliInfo } from './types';
 import { getFullCommand } from './utils';
+import { logger } from '../utils/logger';
 
 const debug = getDebugFunction('twilio-run:start');
 
+type ServerError = Error & {
+  code: string;
+};
+
+function randomPort() {
+  // Returns a random port number higher than 1024 and lower than 65536.
+  return Math.floor(Math.random() * (65535 - 1025) + 1025);
```

Comment:
Should we start by trying out 3001? I know other tools try out 8081 first and if that's available use that one.

Reply:
We agreed that random was cool, but we clamped it between 1025 and 65536.